app/database/models.py

```diff
@@ -30,5 +35,25 @@ class Event(Base):
     start = Column(DateTime, nullable=False)
     end = Column(DateTime, nullable=False)
     owner_id = Column(Integer, ForeignKey("users.id"))
+    owner = relationship("User", back_populates="events") 
+
+    # PostgreSQL
+    if PSQL_ENVIRONMENT:
+        events_tsv = Column(TSVECTOR)
+        __table_args__ = (Index('events_tsv_idx', 'events_tsv', postgresql_using='gin'),)
+
+
+class PSQLEnvironmentError(Exception):
+    pass
+
+
+# PostgreSQL
+if PSQL_ENVIRONMENT:
+    trigger_snippet = DDL("""
+    CREATE TRIGGER ix_events_tsv_update BEFORE INSERT OR UPDATE
```

Comment:
Consider save the query content in other file (probably const)

Reply:
I thought keeping it there would make it easier for other programmers to understand the code, do you think otherwise?

Reply:
I think a good variable name (and maybe a short description) will explain the essence more than the how (the query itself), what do you think?